website/src/pages/download.mdx

```diff
@@ -0,0 +1,40 @@
+import { customFields } from '../../docusaurus.config.json'
+
+export const Link = () => {
+  const curr = `https://github.com/GMOD/jbrowse-components/releases/tag/${customFields.currentVersion}`
+  return <a href={curr}>{curr}</a>
+}
+
+# JBrowse 2 download
+
+To get started with JBrowse 2, we recommend using the "jbrowse CLI"
+
+Run
+
+```
+npm install -g @jbrowse/cli
+jbrowse create jbrowse_folder
+```
+
+This will download the latest jbrowse to a folder called jbrowse_folder
+
+If you have a server with apache or nginx, you can then
```

Comment:
Should also properly capitalize Apache and NGINX

Reply:
Wikipedia says "Nginx, stylized as NGINX, nginx or NginX" so lower case seems allowed